Makefile

```diff
@@ -8,25 +8,22 @@ DESTDIR ?=
 EPOCH_TEST_COMMIT ?= $(shell git merge-base $${DEST_BRANCH:-main} HEAD)
 HEAD ?= HEAD
 
-export PODMAN_VERSION ?= "3.2.0"
+export PODMAN_VERSION ?= "4.0.0"
 
 .PHONY: podman
 podman:
 	rm dist/* || :
-	python -m pip install --user -r requirements.txt
+	$(PYTHON) -m pip install --user -r requirements.txt
 	PODMAN_VERSION=$(PODMAN_VERSION) \
 	$(PYTHON) setup.py sdist bdist bdist_wheel
 
 .PHONY: lint
-lint:
-	$(PYTHON) -m pylint podman || exit $$(($$? % 4));
+lint: tox
+	tox -e black,pylint
```

Comment:
Looks like it can't find tox when you pip install --user tox.  I think it's a PATH problem where $HOME/.local/bin is not in root's path by default.

  WARNING: The script virtualenv is installed in '/root/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
  WARNING: The scripts tox and tox-quickstart are installed in '/root/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.


Seems to work fine locally as non-root.

Reply:
looks like you could update this to "$(PYTHON) -m tox -e black,pylint" instead